### ISSUE TITLE
[Feature request] Edit option for the sticker

### ISSUE THREAD

**Issue by ABedrock** [hm21/pro_image_editor#188]
### Platforms

iOS

### Description

Hi @hm21 ,

Thank you for providing such an excellent image editing platform and for your ongoing maintenance efforts. I greatly appreciate your work.

I'm currently working on a project and need a feature that allows users to change or edit a sticker when they click on it, similar to how text editing works. I've gone through the documentation but couldn't find anything related to this functionality. Could you please guide me on how to implement this feature if it already exists, or consider adding it in a future update? I would be truly grateful.

Thanks

<img width="329" alt="image" src="https://github.com/user-attachments/assets/ddad932e-a1ae-4b8f-8915-b613704dd70d">

<img width="203" alt="image" src="https://github.com/user-attachments/assets/5b6f844f-e7e6-4445-b372-0749657360db">

**Comment by hm21** [comment]
Hi @ABedrock,

Thank you for your kind words. I'm glad to hear that you like my package.

Previously, this was not possible, but I have just released version 5.1.0, which allows you to do that as shown below. The edit button will automatically be visible when you add the callback.

```dart
    ProImageEditor.network(
      _url,
      callbacks: ProImageEditorCallbacks(
        stickerEditorCallbacks: StickerEditorCallbacks(
          onTapEditSticker: (state, sticker, index) async {
            /// Keep in mind that if you use an image, ensure to call precacheImage so that the 
            /// background task takes the correct screenshot.
            state.replaceLayer(
              index: index,
              layer: sticker.copyWith(
                /// Replace Sticker with your updated sticker
                sticker: Sticker(
                  index: newIndex,
                ),
              ),
            );
          },
        ),
      ),
);
```

**Comment by ABedrock** [comment]
I'm glad to have that feature in my app. Thank you!
